File: snippets/ruby/handle_parsed_emails.rb
def handle_inbound_email_webhook(request)
  mandrill_events = request.body[:mandrill_events]
  mandrill_events.each do |event|
    msg = event[:msg]
    from_email = msg[:from_email]
    subject = msg[:subject]
    text = msg[:text]
    
    user = fake_db.get_user_by_email(from_email)
    fake_db.create_to_do({
user: user,
text: subject,
description: text
    })
  end
end

File: snippets/node/handle_parsed_emails.js
function handleInboundEmailWebhook(req) {
  // Get the webhook events from the request body. We know
  // every event will be an inbound event, because we’re
  // only using this endpoint for inbound email.
  const { mandrill_events: inboundEvents } = req.body;

  inboundEvents.forEach(inboundEvent => {
    const {
      msg: { from_email, subject, text }
    } = inboundEvent;

    const user = fakeDb.getUserByEmail(from_email);
    fakeDb.createTodo({
      user,
      text: subject,
      description: text
    });
  });
}